catalogue_graph/src/transformers/catalogue/data/blacklisted_concept_labels.txt

```diff
@@ -0,0 +1,2 @@
+siamese
```

Comment:
I would use the terminology "allowlist" and "denylist" as it's clearer what the purpose is.